pyproject.toml

```diff
@@ -8,8 +8,8 @@ packages = [{include = "ionic_langchain"}]
 
 [tool.poetry.dependencies]
```

Comment:
Do we want to add more metadata here in the effort towards publishing on PyPI?

https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata

Reply:
Eventually 💯